lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -1,8 +1,8 @@
 import hashlib
+import pytest
```

Comment:
This is out of order, because `pytest` isn't standard library. You can run `isort` manually before you commit changes to avoid this sort of thing. (Which will get a bit easier if you enable precommit hooks once Peter's change goes in...)

```bash
pip3 install isort
isort .
```

Reply:
Sure, that makes sense.

Reply:
> This is out of order, because `pytest` isn't standard library. You can run `isort` manually before you commit changes to avoid this sort of thing. (Which will get a bit easier if you enable precommit hooks once Peter's change goes in...)
> 
> ```shell
> pip3 install isort
> isort .
> ```

When I am running `isort` with file name it isn't formatting the imports the way they should be. Especially `pytest` in a separate block.

Reply:
Your branch seems to be up-to-date: or at least I see the correct configuration in `project.toml`. Did you do the `isort` before rebasing to `main`, by any chance?

I've been randomly reorganizing imports in weird orders, and deliberately adding new imports out of order to be sure that `isort` fixes it correctly, and to date I haven't seen any problems.